src/components/ui/button-list.tsx

```
@@ -0,0 +1,114 @@
+const Item = ({
+  title,
+  subTitle,
+  selected = false,
+  className,
+  ...props
+}: ButtonListItemProps) => {
+  return (
+    <motion.li variants={itemVariants}>
+      <button
+        {...props}
+        role='option'
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<option>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)